frontend/packages/policy-editor/src/utils/index.ts

```diff
@@ -141,8 +140,8 @@ export const mapPolicyRuleToPolicyRuleBackendObject = (
       .map((r) => (r.type.startsWith('urn:') ? `${r.type}:${r.id}` : `urn:${r.type}:${r.id}`)),
   );
 
-  const subject: string[] = policyRule.subject.map((s) =>
-    mapSubjectTitleToSubjectString(subjectOptions, s),
+  const subject: string[] = policyRule.subject.map(
+    (s) => s && mapSubjectTitleToSubjectString(subjectOptions, s),
   );
```

Comment:
Can the array of subjects include null or undefined? Is it necessary to check if the value exists before executing the mapSubjectTitleToSubjectString function?

```suggestion
  const subject: string[] = policyRule.subject.map(
    (s) => mapSubjectTitleToSubjectString(subjectOptions, s),
  );
```